### ISSUE TITLE
overview of the exchange - WGBIOP

### ISSUE THREAD

**Issue by kbekaert** [ices-eg/SmartDots#135]
•SmartDots to have a brief overview of the exchange using a standard table to outline the metadata (hoover over the event and some interesting info)

metadata:
Official name of exchange | Species | Stock | Number of countries | Number of Institutes | Number of Readers | No of structures by preparation method

**Comment by wimallegaert** [comment]
integrate in webapplication event view
integrate in smartdots event view (extra prop for web api):
- add number of readers in the event view

**Comment by JCoadDavies** [comment]
@cmspinto  here https://smartdots.ices.dk/manage/ListEvents

**Comment by cmspinto** [comment]
It was only possible to add some of the fields, it is added to an info icon in the beginning:

Official name of exchange | Species | Stock | Number of countries | Number of Readers | No of structures by preparation method

Number of Institutes was not possible to add because readers are not always linked to institutes!

**Comment by cmspinto** [comment]
Should we add some of this to the software?

**Comment by JCoadDavies** [comment]
It looks Carlos but I wonder if we have interpreted Grainnes wishes correctly. I think it would make more sense if **No of structures by preparation method** was the **number of samples** and for this to be by area and preparation method.

**Comment by JCoadDavies** [comment]
> Should we add some of this to the software?

Yes, i especially think the sample overview is useful :-) @wimallegaert @kevinatilvo

**Comment by cmspinto** [comment]
> 
> 
> It looks Carlos but I wonder if we have interpreted Grainnes wishes correctly. I think it would make more sense if **No of structures by preparation method** was the **number of samples** and for this to be by area and preparation method.

Added NoSamples:

![image](https://user-images.githubusercontent.com/8705477/81783758-1d273680-94fc-11ea-8d7f-a67b93e42857.png)

**Comment by JCoadDavies** [comment]
Looks good

**Comment by JaneGodiksen** [comment]
Just a quick question. Is the number of samples per preparation method?
Here there is only one prep.met. but if there would be two methods and 61 fish. This would be 61 samples per prep.met. but 122 images all in all. What result would the "Number of samples" give?

**Comment by cmspinto** [comment]
The number of samples is the total number of samples. 
And the Number of preparation method is how many distinct preparation methods were used in that event.

Goind to the detail of Number of Samples per preparation can be done but only inside the page of the event!
